api/proto/teleport/legacy/types/types.proto

```diff
@@ -993,6 +993,102 @@ message ProvisionTokenSpecV2 {
   ];
 }
 
+// ProvisionTokenV3 specifies provisioning token
+message ProvisionTokenV3 {
+  option (gogoproto.goproto_stringer) = false;
+  option (gogoproto.stringer) = false;
+
+  // Kind is a resource kind
+  string Kind = 1 [(gogoproto.jsontag) = "kind"];
+  // SubKind is an optional resource sub kind, used in some resources
+  string SubKind = 2 [(gogoproto.jsontag) = "sub_kind,omitempty"];
+  // Version is version
```

Comment:
I mean, yes 😅 
Can you add that this is a resource version to make it more "meaningful"?